config/locales/pt-BR.yml

```diff
@@ -25,6 +25,8 @@ pt-BR:
       confirm_message: Você tem certeza?
       export: Exportar
       export_as: Como
+    associations:
+      label: Nested resources
```

Comment:
I was told it's better to leave things untranslated instead of using Google Translate... 😄